db/migrate/20230111234349_add13614_c_page3_columns_to_intakes.rb

```diff
@@ -0,0 +1,8 @@
+class Add13614CPage3ColumnsToIntakes < ActiveRecord::Migration[7.0]
+  def change
+    add_column :intakes, :receive_written_communication, :integer, default: 0, null: false
```

Comment:
@tgrathwell i forget whether we said we were going to base this on the presence of `preferred_written_language`...either way i added it and can easily remove it